### PR TITLE
Support nor Query

### PR DESCRIPTION
**Pull request by dplewis**
https://github.com/parse-community/parse-server/pull/4768

### CODE REVIEW

**Comment by dplewis** [comment]
@flovilmart How does this look?

File: src/ParseQuery.js [hunk 1]
```diff
@@ -1422,6 +1435,24 @@ class ParseQuery {
...
+   * Constructs a Parse.Query that is the NOR of the passed in queries.  For
+   * example:
+   * <pre>const compoundQuery = Parse.Query.nor(query1, query2, query3);</pre>
+   *
+   * will create a compoundQuery that is a nor of the query1, query2, and
+   * query3.
+   * @param {...Parse.Query} var_args The list of queries to AND.
+   * @static
+   * @return {Parse.Query} The query that is the AND of the passed in queries.
+   */
```
Comment (flovilmart) [inline src/ParseQuery.js]: Docs are wrong

File: src/ParseQuery.js [hunk 2]
```diff
@@ -250,6 +249,20 @@ class ParseQuery {
     return this;
   }
 
+  /**
+   * Adds constraint that all of the passed in queries match.
```
Comment (flovilmart) [inline src/ParseQuery.js]: Docs need a change

File: src/ParseQuery.js (hunk 1 above)
Comment (dplewis) [reply src/ParseQuery.js]: Is there a jsdocs checker / linter we can add?

Comment (flovilmart) [reply src/ParseQuery.js]: I mean the text in the docs ;) it says AND as it should says NOR. :)

File: src/ParseQuery.js (hunk 2 above)
Comment (flovilmart) [reply src/ParseQuery.js]: It should say:

‘Adds constraint that none of the passed queries match’

File: src/ParseQuery.js (hunk 1 above)
Comment (dplewis) [reply src/ParseQuery.js]: The PHP SDK has a document checker. In this case the @param is wrong also

Comment (flovilmart) [reply src/ParseQuery.js]: Not sure about that, you should look at them with `npm run docs`.

My point was on the whole documentation that it didn’t match what the function was doing

**Pull request merged**